test/unit/action/info.spec.js

```diff
@@ -50,6 +53,16 @@ describe('Action Info Unit Tests', () => {
       expect(grpc.sendCommand.callCount, 'to equal', 1);
     });
 
+    it('should set percentSynced', async () => {
+      const testTimestamp = new Date().getTime();
+      grpc.sendCommand.withArgs('getInfo').resolves({
+        synced_to_chain: false,
+        best_header_timestamp: testTimestamp / 1000,
+      });
+      await info.getInfo();
+      expect(store.percentSynced, 'to be within', 0, 1);
+    });
+
```

Comment:
This currently breaks the build: https://travis-ci.org/lightninglabs/lightning-app/jobs/403387961#L5266